tlovett1/safe-redirect-manager
# Feature/handle 4xx status codes

### Description of the Change

Handles 403, 404, and 410 status codes. 403 and 410 status codes will add a header and die using `wp_die`. 404 status code will render the 404 template. This will happen even in environments that "soft 404" by default (for example, some environments when "Plain" permalinks are set).

Closes #241 

### How to test the Change

- Create a Redirect Rule that has a status code of 403. On the front end, this request should die with a status code of 403
  - Repeat this step for 410
- Create a Redirect Rule that has a status code of 404. This should render the 404 template with a status code of 404.
  - Repeat this test with permalinks set to "Plain"

### Changelog Entry

> Added - Handling of 403, 404, and 410 status codes

### Credits

Props @nateconley, @aosmichenko

### Checklist:
